components/motor/gpio/controlled.go

```diff
@@ -178,7 +178,8 @@ func (cm *controlledMotor) Stop(ctx context.Context, extra map[string]interface{
 	// after the motor is created, Stop is called, but if the PID controller
 	// is auto-tuning, the loop needs to keep running
 	if cm.loop != nil && !cm.loop.GetTuning(ctx) {
-		cm.loop.Pause()
+		cm.loop.Stop()
+		cm.loop = nil
```

Comment:
I feel like this was kind of the whole point of adding the pause/resume functionality, is there a way to keep this? if not I'm fine with removing it but maybe also look at where else this is being used and remove it if it's no longer useful

Reply:
so the thing I really want is to get blocks to reset() when pause is called. gonna try to figure out if thats a thing I can do

Reply:
tho we do call reset on the pid blocks, so maybe theres a different block we have to reset? or some other state thats not getting reset

Reply:
yeah maybe. If the solution is just to reset all blocks then I think we can go back to creating a new loop every time

Reply:
creating a new loop technically works, but i think I can get away with just redoing the constant blocks. Calling reset on those doesn't actually do what we want

Reply:
I agree with @martha-johnston here, but maybe we added the wrong helper functions and functionality for our drivers to be spammed with SetPoints. 

So, in vanilla controls, I would expect `UpdateConfig` to give me what I want when I request a new SetPoint to Trapezoidal profile. We have ways of storing the state in the summation block too.

Also - it may be worth it to address the encoded motor's issues first and work out a good solution for the control stuff - it could use a more general look since we've been messing with the code so much.